examples/standalone/runfile/dynamics.py

```diff
@@ -1,120 +1,152 @@
+import json
+import pathlib
 from argparse import ArgumentParser
+from datetime import datetime
 
-import mpi4py
+import git
+import numpy as np
 import serialbox
 import yaml
+from mpi4py import MPI
 
 import fv3core
 import fv3core._config as spec
 import fv3core.stencils.fv_dynamics as fv_dynamics
 import fv3core.testing
 import fv3gfs.util as util
-from fv3core.testing import GlobalTimer, write_to_json
+from fv3gfs.util import Timer
+
+
+def print_and_write_global_timings(
+    timer, experiment_name, time_step, backend, comm, root=0
+):
+    is_root = comm.Get_rank() == root
+    recvbuf = np.array(0.0)
+    experiment = {}
+    now = datetime.now()
+    sha = git.Repo(
+        pathlib.Path(__file__).parent.absolute(), search_parent_directories=True
+    ).head.object.hexsha
+    dt_string = now.strftime("%d/%m/%Y %H:%M:%S")
+    filename = now.strftime("%Y-%m-%d-%H-%M-%S")
+    experiment["setup"] = {}
+    experiment["setup"]["experiment time"] = dt_string
+    experiment["setup"]["data set"] = experiment_name
```

Comment:
We normally write "dataset" as one word in Python packages (e.g. netcdf4.Dataset). All the data is for the "experiment", so "experiment time" doesn't really clear things up - it could mean time the experiment took to run. I'd suggest "timestamp" which I think says what you want.

If using spaces I'd suggest underscores instead.

```suggestion
    experiment["setup"]["timestamp"] = dt_string
    experiment["setup"]["dataset"] = experiment_name
```